compiler/rustc_infer/src/infer/error_reporting/mod.rs

```diff
@@ -173,8 +173,10 @@ fn msg_span_from_early_bound_and_free_regions<'tcx>(
                 match fr.bound_region {
                     ty::BoundRegionKind::BrNamed(_, name) => {
                         let mut sp = tcx.def_span(scope);
-                        if let Some(param) =
-                            tcx.hir().get_generics(scope).and_then(|generics| generics.get_named(name))
+                        if let Some(param) = tcx
+                            .hir()
+                            .get_generics(scope)
+                            .and_then(|generics| generics.get_named(name))
```

Comment:
What happened here? did rustfmt give up?

Reply:
Anyways this was just because I manually applied a branch of rustfmt that does format let chains correctly, and it applied this change. I will revert it, since it's not part of the diff.